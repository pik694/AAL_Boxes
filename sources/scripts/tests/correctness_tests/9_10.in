9
69 8 58
73 32 5
43 99 36
75 53 12
39 29 34
18 78 32
64 84 49
62 60 68
59 30 2

