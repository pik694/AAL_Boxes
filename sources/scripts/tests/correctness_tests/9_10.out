8 1 3 6 0 4 7 5 2 
847954->701652
